server/plugin/client.go

```diff
@@ -310,3 +311,21 @@ func (c *client) GetIncidentFieldsFromServiceNow() ([]*serializer.ServiceNowInci
 
 	return fields.Result, statusCode, nil
 }
+
+func (c *client) SearchAssignmentGroupsInServiceNow(searchTerm, limit, offset string) ([]*serializer.ServiceNowAssignmentGroup, int, error) {
+	query := fmt.Sprintf("%s LIKE%s", constants.FieldName, searchTerm)
+	queryParams := url.Values{
+		constants.SysQueryParam:       {query},
+		constants.SysQueryParamLimit:  {limit},
+		constants.SysQueryParamOffset: {offset},
+		constants.SysQueryParamFields: {fmt.Sprintf("%s,%s,%s", constants.FieldSysID, constants.FieldName, constants.FieldDescription)},
```

Comment:
What is the need for "description"?

Reply:
it is fixed in next PR
https://github.com/Brightscout/mattermost-plugin-servicenow/blob/da0904ad8109dcba1cdeefea25b190dd585b8573/server/plugin/client.go#L321